proto/desmos/tokenfactory/v1beta1/params.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package desmos.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v5/x/tokenfactory/types";
+
+// Params defines the parameters for the tokenfactory module.
+//
+// Since: Desmos 6.0.0
+message Params {
+  // DenomCreationFee defines the fee to be charged on the creation of a new
+  // denom. The fee is drawn from the subspace treasruy account, and
+  // transferred to the community pool.
```

Comment:
I don't agree with this implementation choice. Instead, I think the tokens should be burned. This will introduce a deflation event for DSM that will be beneficial for the tokenomics. We should not send those tokens to the community pool.

Reply:
Updated